src/safeposix/syscalls/fs_calls.rs

```diff
@@ -28,10 +26,7 @@ use std::ffi::CString;
 use std::ptr;
 use std::mem;
 
-// use crate::example_grates::vanillaglobal::*;
 use crate::example_grates::dashmapvecglobal::*;
-// use crate::example_grates::muthashmaxglobal::*;
-// use crate::example_grates::dashmaparrayglobal::*;
```

Comment:
Those are required to use different version of fdtables

Reply:
Reverted changes